backend/apps/core/management/commands/generate_typescript_types.py

```
@@ -0,0 +1,64 @@
+        frontend_dir = repo_root / "frontend"
+
+        schema_path = backend_dir / "schema.yml"
+        output_dir = frontend_dir / "src" / "types"
+        output_dir.mkdir(parents=True, exist_ok=True)
+
+        output_file = output_dir / "api.d.ts"
+
+        try:
+            self.stdout.write("Generating OpenAPI schema...")
+
+            subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)